src/config/2020.json

```diff
@@ -571,6 +571,16 @@
       "github": "ericwbailey",
       "twitter": "ericwbailey"
     },
+    "eeeps": {
+      "name": "Eric Portis",
+      "teams": [
+        "authors"
+      ],
+      "avatar_url": "https://avatars1.githubusercontent.com/u/3441390?v=4&s=200",
+      "website": "https://ericportis.com/",
+      "github": "eeeps",
+      "twitter": "etportis"
+    },
```

Comment:
@eeeps please confirm you're happy with this as you didn't have a contributor profile.

Reply:
Works for me!